### PR TITLE
Document blink_features tables

### PR DESCRIPTION
**Pull request by max-ostapenko**

### CODE REVIEW

**Comment by max-ostapenko** [comment]
@tunetheweb please have a look

File: docs/gettingstarted_bigquery.md [hunk 1]
```diff
@@ -89,36 +89,40 @@ Some of the types of tables you'll find useful when getting started are describe
...
     * Lighthouse was intially only run on mobile, but as of May 2021 also runs as part of the desktop crawl.
     * These tables are very large (2.3 TB for Mobile only as of May 2021)
 
 ### Other Tables
 
-* `technologies` tables:
+* [`technologies`](https://console.cloud.google.com/bigquery?ws=!1m4!1m3!3m2!1shttparchive!2stechnologies) tables:
     * Information about the technologies detected on each page (using [Wappalyser rules](https://github.com/HTTPArchive/wappalyzer)).
     * Table contains a url and a list of names and categories for technologies detected on the page.
-    * This data is also available in the HAR of the `pages` table but is extracted into the `technologies` table for easy lookup.
```
Comment (tunetheweb) [inline docs/gettingstarted_bigquery.md:119]: You didn’t like this line?

Comment (max-ostapenko) [reply docs/gettingstarted_bigquery.md:119]: Got lost in sync.

File: docs/gettingstarted_bigquery.md [hunk 2]
```diff
@@ -89,36 +89,41 @@ Some of the types of tables you'll find useful when getting started are describe
...
+* [`technologies`](https://console.cloud.google.com/bigquery?ws=!1m4!1m3!3m2!1shttparchive!2stechnologies) tables:
     * Information about the technologies detected on each page (using [Wappalyser rules](https://github.com/HTTPArchive/wappalyzer)).
     * Table contains a url and a list of names and categories for technologies detected on the page.
     * This data is also available in the HAR of the `pages` table but is extracted into the `technologies` table for easy lookup.
     * These tables are small (15GB as of May 2024).
 
+* [`blink_features.features`](https://console.cloud.google.com/bigquery?ws=!1m5!1m4!4m3!1shttparchive!2sblink_features!3sfeatures) tables:
+    * Information about the [Blink features](https://chromestatus.com/roadmap) detected on each page.
+    * Table contains a url and Blink feature names detected on the page.
+    * These tables are 300GB per single platform as of May 2024.
```
Comment (tunetheweb) [inline docs/gettingstarted_bigquery.md:125]: Will add similar comment here:

```suggestion
    * This data is also available in the HAR of the `pages` table but is extracted into the `blink_features` tables for easy lookup.
    * These tables are 300GB per single platform as of May 2024.
```

File: docs/gettingstarted_bigquery.md [hunk 3]
```diff
@@ -89,36 +89,42 @@ Some of the types of tables you'll find useful when getting started are describe
...
     * Table contains a url and a list of names and categories for technologies detected on the page.
     * This data is also available in the HAR of the `pages` table but is extracted into the `technologies` table for easy lookup.
     * These tables are small (15GB as of May 2024).
 
+* [`blink_features.features`](https://console.cloud.google.com/bigquery?ws=!1m5!1m4!4m3!1shttparchive!2sblink_features!3sfeatures) tables:
+    * Information about the [Blink features](https://chromestatus.com/roadmap) detected on each page.
+    * Table contains a url and Blink feature names detected on the page.
+    * This data is also available in the HAR of the `pages` table but is extracted into the `blink_features` tables for easy lookup.
+    * These tables are 300GB per single platform as of May 2024.
+
```
Comment (tunetheweb) [inline docs/gettingstarted_bigquery.md:127]:
```suggestion

* [`blink_features.usage`](https://console.cloud.google.com/bigquery?ws=!1m5!1m4!4m3!1shttparchive!2sblink_features!3susage) table:
    * Summary information about the [Blink features](https://chromestatus.com/roadmap) detected on each page.
    * Table contains the num_urls, the pct_urls and sample urls for each feature.
    * This data is also available in the HAR of the `pages` table but is extracted into the `blink_features` tables for easy lookup.
    * This table is 944MB as of May 2024.
```

File: docs/gettingstarted_bigquery.md [hunk 4]
```diff
@@ -89,36 +89,42 @@ Some of the types of tables you'll find useful when getting started are describe
...
 
-* `technologies` tables:
+* [`technologies`](https://console.cloud.google.com/bigquery?ws=!1m4!1m3!3m2!1shttparchive!2stechnologies) tables:
     * Information about the technologies detected on each page (using [Wappalyser rules](https://github.com/HTTPArchive/wappalyzer)).
     * Table contains a url and a list of names and categories for technologies detected on the page.
     * This data is also available in the HAR of the `pages` table but is extracted into the `technologies` table for easy lookup.
     * These tables are small (15GB as of May 2024).
 
+* [`blink_features.features`](https://console.cloud.google.com/bigquery?ws=!1m5!1m4!4m3!1shttparchive!2sblink_features!3sfeatures) tables:
+    * Information about the [Blink features](https://chromestatus.com/roadmap) detected on each page.
```
Comment (tunetheweb) [inline docs/gettingstarted_bigquery.md:123]:
```suggestion
    * Information about the [Blink features](https://chromestatus.com/roadmap) detected on each page. See also the  summary `blink_features.usage` table below.
```

File: docs/gettingstarted_bigquery.md [hunk 5]
```diff
@@ -89,36 +89,42 @@ Some of the types of tables you'll find useful when getting started are describe
...
 ### Other Tables
 
-* `technologies` tables:
+* [`technologies`](https://console.cloud.google.com/bigquery?ws=!1m4!1m3!3m2!1shttparchive!2stechnologies) tables:
     * Information about the technologies detected on each page (using [Wappalyser rules](https://github.com/HTTPArchive/wappalyzer)).
     * Table contains a url and a list of names and categories for technologies detected on the page.
     * This data is also available in the HAR of the `pages` table but is extracted into the `technologies` table for easy lookup.
     * These tables are small (15GB as of May 2024).
 
+* [`blink_features.features`](https://console.cloud.google.com/bigquery?ws=!1m5!1m4!4m3!1shttparchive!2sblink_features!3sfeatures) tables:
```
Comment (tunetheweb) [inline docs/gettingstarted_bigquery.md:122]:
```suggestion
* [`blink_features.features`](https://console.cloud.google.com/bigquery?ws=!1m5!1m4!4m3!1shttparchive!2sblink_features!3sfeatures) table:
```

File: docs/gettingstarted_bigquery.md [hunk 6]
```diff
@@ -89,36 +89,42 @@ Some of the types of tables you'll find useful when getting started are describe
...
+    * Information about the [Blink features](https://chromestatus.com/roadmap) detected on each page.
+    * Table contains a url and Blink feature names detected on the page.
+    * This data is also available in the HAR of the `pages` table but is extracted into the `blink_features` tables for easy lookup.
+    * These tables are 300GB per single platform as of May 2024.
```
Comment (tunetheweb) [inline docs/gettingstarted_bigquery.md:126]:
```suggestion
    * This data is also available in the HAR of the `pages` table but is extracted into the `blink_features` tables for easy lookup.
    * This table is ~300GB per single platform as of May 2024.
```

**Review by tunetheweb — APPROVED** [review]
Added the usage table too

**Pull request merged**